src/components/PopoverMenu/BasePopoverMenu.js

```diff
@@ -48,17 +102,24 @@ class BasePopoverMenu extends PureComponent {
                             </Text>
                         </View>
                     )}
-                    {_.map(this.props.menuItems, item => (
-                        <MenuItem
-                            key={item.text}
-                            icon={item.icon}
-                            iconWidth={item.iconWidth}
-                            iconHeight={item.iconHeight}
-                            title={item.text}
-                            description={item.description}
-                            onPress={() => this.props.onItemSelected(item)}
-                        />
-                    ))}
+                    <ArrowKeyFocusManager
+                        focusedIndex={this.state.focusedIndex}
+                        maxIndex={this.props.menuItems.length - 1}
+                        onFocusedIndexChanged={this.updateFocusedIndex}
+                    >
+                        {_.map(this.props.menuItems, (item, menuIndex) => (
+                            <MenuItem
+                                key={item.text}
+                                icon={item.icon}
+                                iconWidth={item.iconWidth}
+                                iconHeight={item.iconHeight}
+                                title={item.text}
+                                description={item.description}
+                                onPress={() => this.props.onItemSelected(item)}
+                                focused={this.state.focusedIndex === menuIndex}
```

Comment:
This prop should be `isFocused` because it's a boolean

Reply:
It wasn't added in this PR. I am using an existing prop. Do you want me to refactor it for `MenuItem`. Might have to check the places where it'll have to be updated.